### PR TITLE
Create 2018-06-29-14-04-netcore-7-0.md

### PR DESCRIPTION
**Pull request by xuzhg**
Add release note for 7.0 RTM

### Issues

* Add release note for 7.0 RTM

### Description

 * Add release note for 7.0 RTM

### Checklist (Uncheck if it is not completed)

- [ ] *Test cases added*
- [ ] *Build and test with one-click build and test script passed*

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+title : "14.4 OData Web API 7.0 now available"
+description: "7.0 release notes"
+category: "14. 7.x Features "
+---
+
+Today we're happy to announce the release of ASP.NET Web API OData 7.0 on the [NuGet gallery](https://www.nuget.org/)!
+
+ASP.NET Web API OData 7.0 is available with two new packages:
+
+1. [Microsoft.AspNetCore.OData](https://www.nuget.org/packages/Microsoft.AspNetCore.OData) is based on ASP.NET Core 2.x.
```
Comment (biaol-odata) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: should we specify 2.0 only, or be more descriptive related to 2.0 vs 2.1?

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title : "14.4 OData Web API 7.0 now available"
+description: "7.0 release notes"
+category: "14. 7.x Features "
+---
+
+Today we're happy to announce the release of ASP.NET Web API OData 7.0 on the [NuGet gallery](https://www.nuget.org/)!
+
```
Comment (madansr7) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: Remove "Today" and just start with "We're happy to announce . . . ."

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+ASP.NET Web API OData 7.0 is available with two new packages:
+
+1. [Microsoft.AspNetCore.OData](https://www.nuget.org/packages/Microsoft.AspNetCore.OData) is based on ASP.NET Core 2.x.
+2. [Microsoft.AspNet.OData](https://www.nuget.org/packages/Microsoft.AspNet.OData) is based on ASP.NET Web API.
+
+[Here](http://odata.github.io/WebApi/#14-01-netcore-beta1) can find beta1, beta2 & beta4 information.
+
```
Comment (madansr7) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: I think we don't need beta pointers in RTM.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 1 above)
Comment (AlanWong-MS) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: I think we should specify the version

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+ 
+ * Both [Microsoft.AspNetCore.OData](https://www.nuget.org/packages/Microsoft.AspNetCore.OData) and [Microsoft.AspNet.OData](https://www.nuget.org/packages/Microsoft.AspNet.OData) are using "**Microsoft.AspNet.OData**" namespace.
+ 
+ * [ [PR #1353](https://github.com/OData/WebApi/pull/1353) ] Change OptionalParameter as Nullable .
+ 
+ * [ [PR #1489](https://github.com/OData/WebApi/pull/1489) ] Change the OptionalReturn to ReturnNullable in OperationConfiguration.
+ 
+ * [ [PR #1409](https://github.com/OData/WebApi/pull/1409) ] Set default to Unqualified-function/acition call and case insensitive.
+
+ * [ [PR #1393](https://github.com/OData/WebApi/pull/1393) ] Set default to enable KeyAsSegment.
```
Comment (biaol-odata) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: this one should be backward compatible.

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  
+<h4>Breaking changes</h4>
+ 
+ * Both [Microsoft.AspNetCore.OData](https://www.nuget.org/packages/Microsoft.AspNetCore.OData) and [Microsoft.AspNet.OData](https://www.nuget.org/packages/Microsoft.AspNet.OData) are using "**Microsoft.AspNet.OData**" namespace.
+ 
+ * [ [PR #1353](https://github.com/OData/WebApi/pull/1353) ] Change OptionalParameter as Nullable .
+ 
+ * [ [PR #1489](https://github.com/OData/WebApi/pull/1489) ] Change the OptionalReturn to ReturnNullable in OperationConfiguration.
+ 
+ * [ [PR #1409](https://github.com/OData/WebApi/pull/1409) ] Set default to Unqualified-function/acition call and case insensitive.
```
Comment (biaol-odata) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: this one should be backward compatible.

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+ * [ [PR #1393](https://github.com/OData/WebApi/pull/1393) ] Set default to enable KeyAsSegment.
+ 
+<h4>Improvements & Fixes:</h4>
+
+ * [ [Issue #1510](https://github.com/OData/WebApi/issues/1510) ] move $top & $skip execute as late as possible.
```
Comment (biaol-odata) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: should the list be sorted by issue#?

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title : "14.4 OData Web API 7.0 now available"
+description: "7.0 release notes"
+category: "14. 7.x Features "
```
Comment (AlanWong-MS) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: Can you double check if the space in `Features "` will impact whether this post will show up? You can compare with the other files.

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title : "14.4 OData Web API 7.0 now available"
```
Comment (AlanWong-MS) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: I think we should take out "now available" due to the style of the docs that we have (in line with Saurabh's comment regarding taking out "today"). If we had a blog, I think the phrase would be more appropriate.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 1 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: #resolved.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 8 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: #resolved.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 7 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: They are same.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 2 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: Removed.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 3 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: Re-added.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 5 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: Moved to fix and performance.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 4 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: Moved to fix and performance.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 6 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: Sorted.

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title : "14.4 OData Web API 7.0"
+description: "7.0 release notes"
+category: "14. 7.x Features "
+---
+
+We're happy to announce the release of ASP.NET Web API OData 7.0 on the [NuGet gallery](https://www.nuget.org/)!
+
+ASP.NET Web API OData 7.0 is available with two new packages:
```
Comment (mikepizzo) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: >new  [](start = 48, length = 4)

Should we use "new" here? the .NETCore is certainly new; the other is a (significant) new release of the 6.x. I'm afraid that "new" makes it sound like there are now 3 options; what they are currently using, 7.0 core, and 7.0 classic.  Maybe change to "ASP.NET WebAPI OData 7.0 is available in two packages:"

Comment (AlanWong-MS) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: +1

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+1. [Microsoft.AspNetCore.OData](https://www.nuget.org/packages/Microsoft.AspNetCore.OData) is based on ASP.NET Core 2.0.
+2. [Microsoft.AspNet.OData](https://www.nuget.org/packages/Microsoft.AspNet.OData) is based on ASP.NET Web API.
+
+[Here](http://odata.github.io/WebApi/#14-01-netcore-beta1) can find beta1, beta2 & beta4 information.
```
Comment (mikepizzo) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: The title is "7.0 Beta1 & Beta2 & Beta4", but this also references 7.0 RTM.  Can we remove references to the individual beta and update this text to be something like "Information on WebAPI OData 7.0 is available [here].  Ideally we would also update section 14.1 to describe the 7.0 release, and remove beta-specific info.

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+PM> Install-Package Microsoft.AspNet.OData
+```
+
+## What’s in this release?
+
+<h4>ASP.NET Core 2.x support</h4>
+
+ See main issues from:
+
+ * [ [Web API issue #939](https://github.com/OData/WebApi/issues/939) ] .NET Core Support - IN Progress
```
Comment (mikepizzo) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: > - IN Progress [](start = 89, length = 14)

Can we remove the "IN Progress"? Makes it sound like .NET Core is not fully supported in this release.

File: _posts/2018-06-29-14-04-netcore-7-0.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+  
+ * [ [Web API issue #772](https://github.com/OData/WebApi/issues/772) ] Roadmap for OData WebAPI run on ASP.NET Core
```
Comment (mikepizzo) [inline _posts/2018-06-29-14-04-netcore-7-0.md]: Isn't this redundant with the above, or does it describe different features/functionality?

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 9 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: #resolved.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 11 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: #Resolved.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 12 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: There are some comments and information in both issues. I think it's easy for customer to go through issues to fetch what they want.

File: _posts/2018-06-29-14-04-netcore-7-0.md (hunk 10 above)
Comment (xuzhg) [reply _posts/2018-06-29-14-04-netcore-7-0.md]: changed.

**Pull request merged**